mw-webapp/src/logic/reportsTable/renderCellItem/renderEditableString.tsx

```diff
@@ -0,0 +1,55 @@
+import {useState} from "react";
+import {EditableText} from "src/component/editableText/EditableText";
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
+
+/**
+ * Cell item props
+ */
+interface EditableStringProps {
+
+  /**
+   * Cell item's text
+   */
+  content: string;
+
+  /**
+   * Row item uuid
+   */
+  rowUuid: string;
+
+  /**
+   * Property name
+   */
+  propertyName: "studentComments" | "learnedForToday";
```

Comment:
why so strange property fo editable string component? Why `renderEditableString` is not a component?

Reply:
Deleted

Reply:
Deleted